README.md

```diff
@@ -29,7 +29,20 @@ steps:
     dotnet-version: '3.1.x' # SDK Version to use; x will use the latest version of the 3.1 channel
 - run: dotnet build <my project>
 ```
+Multiple versions:
+> Note: In case multiple versions are installed, the latest .NET version will be used by default unless another version is specified in the `global.json` file.
 
+```yml
+steps:
+- name: Setup dotnet
+- uses: actions/checkout@v2
+-  uses: actions/setup-dotnet@v1
+  with:
+    dotnet-version: | 
+      5.0.402
+      3.1.201
```

Comment:
Final nit: I think we should keep the `.x` wildcards to keep consistency with the rest of the readme:
```
    dotnet-version: | 
      3.1.x
      5.0.x
```

Reply:
updated 👍🏾